kubernetes/samples/scripts/create-soa-domain/create-domain.sh

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env bash
```

Comment:
Is this substantially similar to an existing create-domain script?  If so, can it be consolidated and moved to a common location?

Reply:
The cut-and-paste proliferation of this script is a problem.   The fact that create-fmw-infrastructe-domain is doing it too makes the problem worse @anpanigr told me he's going to look into this.